### PR TITLE
Implement replaceSKU method

### PR DESCRIPTION
**Pull request by koz**
Implement replaceSKU for SAC feature in checkout-ui

### CODE REVIEW

File: src/checkout.coffee [hunk 1]
```diff
@@ -436,6 +436,31 @@ class Checkout
...
+      }
+    ]
+
+    @_updateOrderForm({
+      url: @_getAddToCartURL()
+      type: 'PATCH'
+      data: JSON.stringify({
+        "orderItems": items,
+        "expectedOrderFormSections": expectedOrderFormSections,
+        "noSplitItem": true,
```
Comment (brenoc) [inline src/checkout.coffee]: That should be an option

Comment (koz) [inline src/checkout.coffee]: The entire data object?

Comment (koz) [inline src/checkout.coffee]: Or only `noSplitItem`?

**Comment by brenoc** [comment]
Only noSplitItem

Em Sáb, 27 de mai de 2017 19:25, Gabriel Kozlowaski <
notifications@github.com> escreveu:

**Comment by koz** [comment]
@brenoc Updated docs, and passed `items` and `noSplitItem` as argument.

**Pull request merged**